tools/downloader/license.txt

```diff
@@ -3177,7 +3177,7 @@ License terms:
 
 ==================================================================================================
 
-* densenet-121-tf, densenet-169-tf, densenet-161-tf - Tensorflow implementation of DenseNet by G. Huang, Z. Liu, K. Weinberger, and L. van der Maaten with ImageNet pretrained models.
+* densenet-121-tf, densenet-169-tf, densenet-161-tf - TensorFlow implementation of DenseNet by G. Huang, Z. Liu, K. Weinberger, and L. van der Maaten with ImageNet pretrained models.
```

Comment:
do we no need to add asterix?

Reply:
I'd rather keep this PR focused on capitalization. We can go on an asterisk hunt another time.

Reply:
okey